recipe/meta.yaml

```
@@ -6,68 +6,92 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/nannyml-{{ version }}.tar.gz
-  sha256: db0e4a688d9d830ad3c461106f42335588ee03241f16a00fde00767b408ef704
+  url: https://github.com/NannyML/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 67801824c003ec37b055a6c0f4d540e20d3eff8f84cb6e202a40d2b349c092df
 
 build:
+  number: 0
   entry_points:
     - nannyml=nannyml.cli.cli:cli
     - nml=nannyml.cli.cli:cli
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # https://github.com/NannyML/nannyml/blob/v0.12.1/pyproject.toml#L33
+  skip: True  # [py<38 or py>311]
+  # nothing provides requested flaml >=1.0.11,<2.0 on s390x
+  # nothing provides requested python-kaleido 0.2.1.* on s390x
+  skip: True  # [s390x]
 
 requirements:
   host:
     - pip
     - poetry-core >=1.0.0
-    - python >=3.9,<3.12
+    - python
   run:
-    - python >=3.9,<3.12
-    - pandas >=1.5.0,<2.0.0
+    - python
+    - pandas >=1.5,<2
     - python-dateutil >=2.8.2,<3.0.0
-    - scikit-learn >=1.2,<2.0.0
-    - joblib >=1.1.0,<2.0.0
+    - scikit-learn >=1.2,<2.0
+    - joblib >=1.1.0,<2.0
     - category_encoders >=2.3.0,<3.0.0
-    - matplotlib-base >=3.7.0,<4.0.0
+    - matplotlib >=3.7,<4.0
```

```suggestion
    - matplotlib-base >=3.7,<4.0
```
matplotlib-base is better because it doesn't have pyqt